packages/core/test/overlay/overlayTests.tsx

```diff
@@ -333,17 +336,20 @@ describe("<Overlay>", () => {
             assertFocus("button", done);
         });
 
-        function assertFocus(selector: string, done: MochaDone) {
-            wrapper.update();
-            // the behavior being tested relies on requestAnimationFrame. to
-            // avoid flakiness, use nested setTimeouts to delay execution until
-            // the next frame, not just to the end of the current frame.
+        function assertFocus(selector: string | (() => void), done: MochaDone) {
+            // the behavior being tested relies on requestAnimationFrame.
+            // to avoid flakiness, use nested setTimeouts to delay execution until the *next* frame.
```

Comment:
this is still concerning, but at least this PR leaves us in a better state than the repo currently is...

Reply:
yes, it's definitely a better state. in the course of development, i saw test flakes locally with chrome headless where certain suites would go red for a run. but this hasn't shown itself on circle (greens all the way down), so that's good news.